### PR TITLE
Messaging pydantic

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Update workflow helpers to use latest version of messaging with type hinting.

Question: What are the ramifications of no longer being able to provide application stack and runtime environment in this module?
Do these requests need to go through the UserAPI now?

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -15,3 +15,22 @@ presubmits:
...
               memory: "2Gi"
               cpu: "2"
+  - name: thoth-mypy-py38
+    decorate: true
+    skip_report: false
+    always_run: true
+    context: aicoe-ci/prow/mypy
+    spec:
+      containers:
+        - image: quay.io/thoth-station/thoth-pytest-ubi8-py38:v0.12.6
```
Comment (harshad16) [inline .prow.yaml:25]:
```suggestion
        - image: quay.io/thoth-station/thoth-pytest-py38:v0.12.6
```

**Review by pacospace — APPROVED** [review]
lgtm! 💯

**Pull request merged**